app/lib/authorizer.rb

```diff
@@ -98,6 +98,7 @@ def authorized_when_viewing_as(view_as_educator, &block)
   def is_authorized_for_student?(student, options = {})
     begin
       return true if @educator.districtwide_access?
+      return true if @educator.labels.include?('high_school_house_master') && student.grade == '8' && ENV['HOUSEMASTERS_AUTHORIZED_FOR_GRADE_8']
```

Comment:
There are currently no `"high_school_house_master"` labels in the database, so this auth feature can be manually QA'ed in production before we turn it on for housemasters.